res/controllers/Traktor-Kontrol-S4-MK3.js

```diff
@@ -1128,17 +1128,15 @@ class Mixer extends ComponentContainer {
                 switch (value) {
                 case 0x00:  // Picnic Bench / Fast Cut
                     engine.setValue("[Mixer Profile]", "xFaderMode", 0);
-                    engine.setValue("[Mixer Profile]", "xFaderCalibration", 0.9);
                     engine.setValue("[Mixer Profile]", "xFaderCurve", 7.0);
                     break;
                 case 0x01:  // Constant Power
                     engine.setValue("[Mixer Profile]", "xFaderMode", 1);
-                    engine.setValue("[Mixer Profile]", "xFaderCalibration", 0.3);
+                    engine.setValue("[Mixer Profile]", "xFaderCalibration", 0.31498);
```

Comment:
A short comment on how the "magic" number `0.31498` is calculated would be great :)

Reply:
yes, `EngineXfader::getPowerCalibration()`
comment added